DataFormats/CTPPSDigi/src/CTPPSPixelDataError.cc

```diff
@@ -0,0 +1,98 @@
+
+#include "DataFormats/CTPPSDigi/interface/CTPPSPixelDataError.h"
+
+
+CTPPSPixelDataError::CTPPSPixelDataError() {}
+
+CTPPSPixelDataError::CTPPSPixelDataError(uint32_t errorWord32, const int errorType, int fedId) : 
+  errorWord32_(errorWord32),
```

Comment:
please initialize errorWord64_
(and errorWord32_ in the other constructor)
so that written data doesn't have random uninitialized values

Reply:
done